app/src/main/java/ca/llamabagel/transpo/search/ui/SearchViewModel.kt

```diff
@@ -26,7 +28,9 @@ class SearchViewModel @Inject constructor(
     private val updateQuery: UpdateQueryUseCase
 ) : ViewModel() {
 
-    private val _keyboardState = MutableLiveData<KeyboardState>().apply { value = KeyboardState.OPEN }
+    private val _keyboardState = MutableLiveData<KeyboardState>().apply { value =
+        KeyboardState.OPEN
+    }
```

Comment:
Why does this have to be opened up if `_searchResults` (line 36) doesn't?

Reply:
I guess it's also bad form to have a single line `apply`?

Reply:
This was done by the ktlint formatter, so I don't know.
single line `apply` is fine by me, I think ktlint just has a shorter interpretation of what a line is.